<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1529</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabMenu">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>1371</width>
      <height>941</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="filePathLineEdit1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select a file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton1">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>22</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditZone">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>311</width>
        <height>491</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Load a file, then edit it if needed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createFileButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>580</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create a new file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="captureButton">
      <property name="geometry">
       <rect>
        <x>244</x>
        <y>580</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Capture Inputs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>580</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh preview</string>
      </property>
     </widget>
     <widget class="QLabel" name="recordingLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>620</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Not currently recording inputs</string>
      </property>
     </widget>
     <widget class="QPushButton" name="emptyButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>630</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Empty Current File</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLineEdit" name="filePathLineEdit2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select a file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QComboBox" name="appComboBox">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>360</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="replayLabel">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>290</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Which app will you want to&lt;br&gt; replay your inputs in ?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton3">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>370</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="replayButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>630</y>
        <width>361</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Replaying</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1529</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufILE">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menufILE"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
